internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/AbstractTestVeniceHelixAdmin.java

```diff
@@ -263,9 +246,9 @@ VeniceHelixAdmin getFollower(List<VeniceHelixAdmin> admins, String cluster) {
   /**
    * Participant store should be set up by child controller.
    */
-  void verifyParticipantMessageStoreSetup() {
+  private void verifyParticipantMessageStoreSetup() {
     String participantStoreName = VeniceSystemStoreUtils.getParticipantStoreNameForCluster(clusterName);
-    TestUtils.waitForNonDeterministicAssertion(5, TimeUnit.SECONDS, () -> {
+    TestUtils.waitForNonDeterministicAssertion(10, TimeUnit.SECONDS, () -> {
```

Comment:
Any reason to not keep it 5? Just wondering if this will increase the tests times and if this necessary

Reply:
This is to keep parity with removed code in the `setupCluster`. Although, I think 5 seconds should also be okay. I can try reverting it.

The runtime increase is not significant to lead to a increase of overall test time. Only in failure cases will it go till 10 seconds. If the condition is met earlier than 5 seconds, there will be no impact to the runtime.